components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3429,11 +3449,34 @@ std::string RewardsServiceImpl::GetExternalWalletType() const {
           country_codes::CountryCharsToCountryID(country.at(0), country.at(1));
 
       if (id == current_country)
-        return ledger::constant::kWalletBitflyer;
+        return true;
     }
   }
+  return false;
+}
+
+std::string RewardsServiceImpl::GetExternalWalletType() const {
+  if (IsBitFlyerRegion()) {
+    return ledger::constant::kWalletBitflyer;
+  }
+#if BUILDFLAG(ENABLE_GEMINI_WALLET)
+  if (base::FeatureList::IsEnabled(features::kGeminiFeature)) {
+    return profile_->GetPrefs()->GetString(prefs::kExternalWalletType);
+  }
+#endif
 
   return ledger::constant::kWalletUphold;
 }
 
+void RewardsServiceImpl::SetExternalWalletType(const std::string wallet_type) {
+#if BUILDFLAG(ENABLE_GEMINI_WALLET)
```

Comment:
`const std::string&`

Reply:
you mean for this line https://github.com/brave/brave-core/pull/8566/files?file-filters%5B%5D=.cc&file-filters%5B%5D=.h#diff-b3817ac19a1c6ad3db2832fc878795ccc8a6f5fc0ae281cdf70d85a3a0a3d324R3471 right?

Reply:
fixed - https://github.com/brave/brave-core/pull/8566/files#diff-b3817ac19a1c6ad3db2832fc878795ccc8a6f5fc0ae281cdf70d85a3a0a3d324R3471